src-tauri/src/commands/groups.rs

```diff
@@ -501,6 +497,55 @@ pub async fn send_mls_message(
     Ok(inner_event)
 }
 
+/// Creates an unsigned nostr event with the given parameters
+async fn create_unsigned_nostr_event(
+    nostr_keys: &Arc<dyn NostrSigner>,
+    message: String,
+    kind: u16,
+    tags: Option<Vec<Tag>>,
+) -> Result<UnsignedEvent, Error> {
+    let mut final_tags = tags.unwrap_or_default();
+    final_tags.extend(bolt11_invoice_tags(&message));
+
+    let mut inner_event = UnsignedEvent::new(
+        nostr_keys.get_public_key().await?,
+        Timestamp::now(),
+        kind.into(),
+        final_tags,
+        message,
+    );
+    inner_event.ensure_id();
+    Ok(inner_event)
+}
+
+/// Parses a message for BOLT11 invoices and returns corresponding tags
+fn bolt11_invoice_tags(message: &str) -> Vec<Tag> {
+    let mut tags = Vec::new();
+
+    // Check if message contains what looks like a bolt11 invoice
+    if let Some(word) = message
+        .split_whitespace()
+        .find(|w| w.to_lowercase().starts_with("lnbc"))
```

Comment:
[req] we want to support different networks of bolt11. At least the Bitcoin related ones `lntb` `lntbs` `lnbcrt`. The last one we actually support it  
https://github.com/lightning/bolts/blob/master/11-payment-encoding.md#human-readable-part

Reply:
done!